src/packages/core/workspace/components/workspace-split-view/workspace-split-view-variant-selector.element.ts

```diff
@@ -209,10 +209,11 @@ export class UmbWorkspaceSplitViewVariantSelectorElement extends UmbLitElement {
 	}
 
 	override render() {
+		// TODO: Localize [NL]
 		return html`
 			<uui-input
 				id="name-input"
-				label="Document name (TODO: Localize)"
+				label="Document name"
```

Comment:
This component is reused across the Document, Document Blueprint, Media and Members workspaces, so I'm thinking a generic "Enter a name..." label might be best? We already have a localization key for it... `#placeholders_entername`.

I'm happy to update the PR. 👍